CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+set(CMAKE_SKIP_TEST_ALL_DEPENDENCY FALSE)
```

Comment:
"Added in version 3.29." - good news!

Which is fine by me, but should we bump the minimum?